src/specification/endpoints/batchUpload/getBatchUpload/response.json

```diff
@@ -0,0 +1,22 @@
+{
+  "description": "this is a desc for request",
```

Comment:
no it's the response :-) shall we get rid of those dummy descriptions or have a generic placeholder? this issue is in all new responses